plugins/plugin-openwhisk/src/lib/cmds/openwhisk-core.ts

```diff
@@ -1379,7 +1343,7 @@ const makeInit = (commandTree) => async (isReinit = false) => {
         const entities = (synonymsTable.entities[api] || []).concat([api])
         const verbs = synonymsTable.verbs[verb] || []
         entities.forEach(eee => {
-          commandTree.subtreeSynonym(`/wsk/${eee.nickname || eee}`, apiMaster, { usage: docs(eee.nickname || eee) })
```

Comment:
I removed the `usage` here for the following reasons:
1. The type of `commandTree.subtreeSynonym` doesn't support the command option as the third parameter here.
2. The `usage` seems duplicated because the second parameter `apiMaster` already contains the command option with a usage